test/functional/counter-test.js

```diff
@@ -0,0 +1,63 @@
+import { assert } from 'chai';
+import sinon from 'sinon';
+import { Dispatcher } from '../../.dist/index';
+import CounterStore from './counter-store';
+
+describe('Counter', () => {
+
+    let spyA = sinon.spy();
+    let spyB = sinon.spy();
```

Comment:
What is the point of having 2 spies? Looks like you can remove spyB and still have the same test results.